packages/clay-color-picker/src/Editor.tsx

```diff
@@ -192,17 +219,13 @@ export function Editor({
 				</div>
 			</div>
 
-			<Hue
-				onChange={(hue) => {
-					if (hue < LimitValue.min) {
-						hue = LimitValue.min;
-					} else if (hue > LimitValue.maxHue) {
-						hue = LimitValue.maxHue;
-					}
-					onHueChange(hue);
-					onColorChange(tinycolor({h: hue, s, v}));
+			<Alpha
+				color={`#${color.toHex()}`}
+				onChange={(value: number) => {
+					const newColor = color.clone();
+					onColorChange(newColor.setAlpha(value));
 				}}
-				value={hue}
+				value={alpha}
```

Comment:
Maybe we can get the alpha from the color too.

```suggestion
				value={color.getAlpha()}
```